packages/shared/src/lib/constants.ts

```diff
@@ -67,6 +67,7 @@ export const managePlusUrl = 'https://r.daily.dev/billing';
 export const plusDetailsUrl = 'https://r.daily.dev/plus-onboarding';
 export const plusSuccessUrl = `${plusUrl}/success`;
 export const walletUrl = `${webappUrl}wallet`;
+export const settingsUrl = `${webappUrl}settings/`;
```

Comment:
feels off with `/` at the end?

Reply:
yes and no 🙈
Because `webappUrl` has a trailing slash, we usually do `${webappUrl}some-page`, so with trailing slash in `settingsUrl`, we can follow the same usage.

Reply:
it is just, someone uses `settingsUrl` by itself for any reason it would have trailing slash 🤷

Reply:
Reverted the trailing slash